src/internal/easee/connector.go

```diff
@@ -52,7 +46,7 @@ func (c *connector) Disconnect(_ adapter.Thing) {
 }
 
 func (c *connector) Connectivity() *adapter.ConnectivityDetails {
-	if c.signalRClient.Connected() {
+	if c.manager.Connected() {
```

Comment:
This will need to be improved, so the algorithm is based on a signalR connection state + subscription status of a particular charger, not a signalR client in general, but it's not the scope of this task